GeneratorInterface/Core/interface/BaseHadronizer.h

```diff
@@ -111,6 +111,8 @@ namespace gen {
     std::vector<double> randomInitWeights_;
     std::vector<std::string> randomInitConfigDescriptions_;
     std::vector<std::string> gridpackPaths_;
+
+    char *args[5];
```

Comment:
Hi @swayand- why is this change needed? It looks like this variable is used in only one spot.